lib/manager.js

```diff
@@ -309,6 +315,9 @@ class Manager extends EventEmitter {
     ev.on('exec:flags:set', onSetFlags.bind(this, this, { id }))
     ev.on('exec:ping', onPingEvent.bind(this, this, id))
 
+    // auth
+    ev.on('auth:token:updated', this._onAuthTokenUpdated.bind(this))
```

Comment:
`auth:token:updated` is an internal Bitfinex event? Where is this documented?

Reply:
it's a plugin-generated event, check the other MR. I'm also accepting suggestions of better ways to publish/consume it.

Reply:
Maybe changing the `notifyPlugin` function and pass down the ref for the manager instance, so I can publish it directly. WDYT?

Reply:
no, I think an event is better for low coupling, but need to be well documented

Reply:
but.. if `auth:token:updated` is a plugin specific event, then the core depends on its plugin. @robertkowalski what do you think?

Can the plugin directly call the commands from `_onAuthTokenUpdated` (looks like only one `auth` method)?

Reply:
done, I've moved the call to the plugin